app/bundles/AssetBundle/Views/Asset/form.html.php

```diff
@@ -13,6 +13,9 @@
     $view['translator']->trans('mautic.asset.asset.menu.new');
 $view['slots']->set("headerTitle", $header);
 $view['slots']->set('mauticContent', 'asset');
+
+echo $view['assets']->buildAddonScript('bundlename/assets/assets.js');
```

Comment:
Yep that's what I was hinting at in my comment above :p Don't know how that happened - I'll fix it.